examples/src/main/java/tachyon/examples/BasicNonByteBufferOperations.java

```diff
@@ -37,20 +44,30 @@
  * </p>
  * <p>
  * This example also let users play around with how to work with files a bit more. The
- * {@link tachyon.client.ReadType} is something that can be set, as well as ability to delete file
- * if exists.
+ * {@link tachyon.client.TachyonStorageType} is something that can be set, as well as ability to
+ * delete file if exists.
  * </p>
  */
 public final class BasicNonByteBufferOperations implements Callable<Boolean> {
+  // private access to the reinitializer of ClientContext
+  private static ClientContext.ReinitializerAccesser sReinitializerAccesser =
+      new ClientContext.ReinitializerAccesser() {
+        @Override
+        public void receiveAccess(ClientContext.PrivateReinitializer access) {
+          sReinitializer = access;
+        }
+      };
+  private static ClientContext.PrivateReinitializer sReinitializer;
   private final TachyonURI mMasterLocation;
   private final TachyonURI mFilePath;
-  private final WriteType mWriteType;
-  private final ReadType mReadType;
+  private final TachyonStorageType mWriteType;
+  private final TachyonStorageType mReadType;
   private final boolean mDeleteIfExists;
   private final int mLength;
 
   public BasicNonByteBufferOperations(TachyonURI masterLocation, TachyonURI filePath,
-      WriteType writeType, ReadType readType, boolean deleteIfExists, int length) {
+      TachyonStorageType writeType, TachyonStorageType readType, boolean deleteIfExists,
```

Comment:
@calvinjia Actually here I make both readType& writeType as TachyonStorageType, because according to original example, they referred to read&write option in tachyon layer, so UnderStorageType is always STORE in new example.